src/test/java/org/springframework/integration/aws/s3/config/xml/AmazonS3OutboundChannelAdapterParserAWSTests.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2002-2014 the original author or authors.
```

Comment:
Since this is a new class, the date must be only `2015`.
Minor, of course, and will be addressed on the merge.